### DISCUSSION TITLE
Finding all the outdated SCT mappings

### DISCUSSION THREAD

**Discussion by linforestzhang** [DiseaseOntology/HumanDiseaseOntology#1264] [General]
Hi Allen.  I'm using Python and West Coast Informatics Terminology API to help find all the outdated SCT mappings (much more than I expected).  Later the code list will share with you and Dr. Lynn when finished.  Github link is not stable for me.

**Comment by allenbaron** [comment]
Thanks! I'm not familiar with West Coast Informatics. Do you use their services a lot?

We recently discovered that the software we use for UMLS updates has stopped updating some cross-references. It's not clear how long it's had this issue but we are planning to correct it before we incorporate the next UMLS update, which should be some time early next year.

**Comment by linforestzhang** [comment]
It's my first time to use WCI API.  It was found by MS Bing search and pretty easy to use the RESTful API although I'm not familiar with it.
